libr/include/r_vector.h

```diff
@@ -65,6 +65,7 @@ R_API void r_vector_clear(RVector *vec);
 
 // frees the vector and calls vec->free on every element if set.
 R_API void r_vector_free(RVector *vec);
+R_API void r_vector_fini(RVector *vec);
```

Comment:
Not necessary, `r_vector_clear()` does the same thing. You could make fini an alias though, to make it more intuitive.

Reply:
what about calling it r_vector_fini and make an alias named r_vector_clear? because having init without fini makes no sense

Reply:
Ok for me too